packages/circuit-ui/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # @sumup/circuit-ui
 
+## 5.0.0
+
+### Minor Changes
+
+- [#1359](https://github.com/sumup-oss/circuit-ui/pull/1359) [`5804fd8d`](https://github.com/sumup-oss/circuit-ui/commit/5804fd8d720a7594bb880b9956a3310609cfbaa9) Thanks [@amelako](https://github.com/amelako)! - Added a `z-index` to the NotificationToasts so they always float above other content. Requires updating to `@sumup/design-tokens@3.3.0`.
+
+### Patch Changes
+
+- Updated dependencies [[`5804fd8d`](https://github.com/sumup-oss/circuit-ui/commit/5804fd8d720a7594bb880b9956a3310609cfbaa9)]:
+  - @sumup/design-tokens@3.3.0
```

Comment:
We need to remove this

Reply:
Why? I think it's helpful.

Reply:
I would leave it too.

Reply:
It's not a patch change, and it duplicates the other changelog entry. I've been removing them until now but don't have string opinions about it 🤷